src/autolabel/models/hf_pipeline.py

```diff
@@ -13,7 +13,17 @@ class HFPipelineLLM(BaseModel):
 
     def __init__(self, config: AutolabelConfig, cache: BaseCache = None) -> None:
         try:
-            from transformers import AutoModelForSeq2SeqLM, AutoTokenizer, pipeline
+            from transformers import (
+                AutoConfig,
+                AutoModelForSeq2SeqLM,
+                AutoModelForCausalLM,
+                AutoTokenizer,
+                pipeline,
+            )
+            from transformers.models.auto.modeling_auto import (
+                MODEL_FOR_CAUSAL_LM_MAPPING,
```

Comment:
this is really great that this exists in huggingface itself! We don't need to update this ourselves. Thanks for finding this